docs/video/obs/hotkeys.md

```diff
@@ -2,23 +2,25 @@
 icon: obs/hotkeys
 ---
 
-!!! info "All the keybinds on this page are what I've set, <u>OBS has no default keybinds</u>"
 
 If you put the same keybind on a start X and stop X / enable X and disable X it will work like a toggle.
 
 
+* `Enable Preview` & `Disable Preview`
 
-* `Enable Preview` & `Disable Preview`: <kbd>CTRL+ENTER</kbd>
+:   This helps with performance, I like to disable it after starting recording.
 
-:   This may help with performance, I like to disable it after starting recording.
+<!--
+It definitely does help with performance - no need for 'may'
+-->
 
-* `Reset Stats`: <kbd>CTRL+SHIFT</kbd>
```

Comment:
why remove the suggested hotkey?

Reply:
suggested hot keys is like suggesting someone to use specific keybinds in games; it's personal preference. and you already put save replay as "None by default" which is true for all of them (you even mentioned it)